Integrations/QRadar/QRadar.py

```diff
@@ -1038,6 +1048,116 @@ def get_domains_by_id_command():
         }
 
 
+def upload_indicators_list_request(reference_name, indicators_list):
+    """
+        Upload indicators list to the reference set
+
+        Args:
+              reference_name (str): Reference set name
+              indicators_list (List): Indicators values list
+        Returns:
+            Dict. Reference set object
+    """
+    url = '{0}/api/reference_data/sets/bulk_load/{1}'.format(SERVER, urllib.quote(reference_name, safe=''))
+    params = {'name': reference_name}
+    return send_request('POST', url, params=params, data=json.dumps(indicators_list))
+
+
+def upload_indicators_command():
+    """
+        The function finds indicators according to user query and updates QRadar reference set
+    """
+    try:
+        args = demisto.args()
+        reference_name = args.get('ref_name')
+        element_type = args.get('element_type')
+        if not check_ref_set_exist(reference_name):
+            if element_type:
+                create_reference_set(reference_name, args.get('element_type'), args.get('timeout_type'),
```

Comment:
Not sure that this a good behaviour, do we have another command to create ref set? if so just error out and point the user to use it, otherwise a simple typo will cause creating a new one